assets/src/css/admin/setup.scss

```diff
@@ -0,0 +1,135 @@
+/**
+ * Give Setup SCSS
+ *
+ * @description Setup Page Styles
+ * @package     Give
+ * @subpackage  SCSS/Admin
+ * @copyright   Copyright (c) 2020, GiveWP
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+*/
+
+@import url(https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap);
+
+section,
+section * {
+	font-family: 'Open Sans', sans-serif;
+}
+
+section {
+	margin: 20px;
+	margin-bottom: 40px;
+	color: #424242;
+	background-color: #fff;
+	box-shadow: 0 2px 4px rgba(0, 0, 0, 0.09);
+	border-radius: 6px;
+}
+header {
+	padding: 0 40px 0 20px;
+	font-size: 22px;
+	line-height: 22px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	border-bottom: 1px solid #ddd;
+}
+footer {
+	font-size: 16px;
+	padding: 20px;
+	display: flex;
+	justify-content: space-between;
+}
+footer a {
+	color: #0073aa;
+	text-decoration: none;
+}
+footer a::after {
+	content: ' >';
```

Comment:
Let's use the FontAwesome chevron here instead of a > character. Currently, it looks inconsistent with the rest of the page.

Reply:
Ah, sure. The design used a text character, so I didn't think about it. Good catch!